tests/test_main_content.py

```diff
@@ -116,6 +117,51 @@ def test_build(self):
         self.assertTrue(len(listing) == 1)
         self.assertEqual(listing[0]["rsconnect_build_status"], BuildStatus.COMPLETE)
 
+    def test_build_retry(self):
+        connect_server = require_connect()
+        api_key = require_api_key()
+        runner = CliRunner()
+
+        # add a content item
+        args = ["content", "build", "add", "-g", _content_guids[0]]
+        apply_common_args(args, server=connect_server, key=api_key)
+        result = runner.invoke(cli, args)
+        self.assertEqual(result.exit_code, 0, result.output)
+        self.assertTrue(
+            os.path.exists("%s/%s.json" % (_test_build_dir, _normalize_server_url(os.environ.get("CONNECT_SERVER"))))
+        )
+
+        # list the "tracked" content
+        args = ["content", "build", "ls", "-g", _content_guids[0]]
+        apply_common_args(args, server=connect_server, key=api_key)
+        result = runner.invoke(cli, args)
+        self.assertEqual(result.exit_code, 0, result.output)
+        listing = json.loads(result.output)
+        self.assertTrue(len(listing) == 1)
+        self.assertEqual(listing[0]["guid"], _content_guids[0])
+        self.assertEqual(listing[0]["bundle_id"], "176")
+        self.assertEqual(listing[0]["rsconnect_build_status"], BuildStatus.NEEDS_BUILD)
+
+        # set the content build status to RUNNING so it looks like it was interrupted
+        # and the cleanup did not have time to finish, otherwise it would be marked as ABORTED
+        store = ContentBuildStore(RSConnectServer(connect_server, api_key))
+        store.set_content_item_build_status(_content_guids[0], BuildStatus.RUNNING)
+
+        # run the build
```

Comment:
I don't think its a blocker or anything, but you could also add some tests for the other hidden flags that are considered when a retry is executed if you wanted to.